src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -8,16 +8,15 @@ import mesosphere.marathon.api.MarathonRestModule
 import mesosphere.chaos.AppConfiguration
 import mesosphere.marathon.event.{ EventModule, EventConfiguration }
 import mesosphere.marathon.event.http.{ HttpEventModule, HttpEventConfiguration }
-import com.google.inject.AbstractModule
+import com.google.inject.{ Module, AbstractModule }
```

Comment:
getEventsModule uses AbstractModule. Guess this is not needed any longer and we should consistently use Module.